cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -378,11 +378,32 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 		klog.V(4).Infof("Calculating unneeded nodes")
 
 		scaleDown.CleanUp(currentTime)
-		potentiallyUnneeded := getPotentiallyUnneededNodes(autoscalingContext, allNodes)
+
+		var scaleDownCandidates []*apiv1.Node
+		var harborNodes []*apiv1.Node
+
+		if a.processors == nil || a.processors.ScaleDownNodeProcessor == nil {
+			scaleDownCandidates = allNodes
+			harborNodes = allNodes
+		} else {
+			var err errors.AutoscalerError
+			a.processors.ScaleDownNodeProcessor.Reset()
+			scaleDownCandidates, err = a.processors.ScaleDownNodeProcessor.GetScaleDownCandidates(
+				autoscalingContext, allNodes)
+			if err != nil {
+				klog.Error(err)
```

Comment:
I'd prefer to log some information about where the error happened.

Reply:
Line number should help in main.go